sdk/personalizer/ai-personalizer-rest/samples-dev/rankActionsAndrewardEvents.ts

```diff
@@ -0,0 +1,92 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+/**
+ * @summary Demonstrates the use of a Personalizer client to rank actions and reward the presented action.
+ */
+import Personalizer, {
+  isUnexpected,
+  PersonalizerErrorOutput,
+  RankableAction,
+  RankRequest,
+} from "@azure-rest/ai-personalizer";
+
+// Load the .env file if it exists
+import * as dotenv from "dotenv";
+dotenv.config();
+
+async function main() {
+  const endpoint = process.env.PERSONALIZER_ENDPOINT || "<endpoint>";
+  const key = process.env.PERSONALIZER_API_KEY || "<test-key>";
+
+  const client = Personalizer(endpoint, { key: key });
```

Comment:
nit: if the variable (key) has the same name as the property (key) you can do this 

```ts
const client = Personalizer(endpoint, { key });
```